meta-lmp-bsp/recipes-sota/fioefi/fioefi/tegra/fioefi-soc.sh.in

```diff
@@ -1,6 +1,6 @@
 CAPSULE_FILE="tegra-bl.cap"
 CAPSULE_PATH="${FIO_OSTREE_TARGET_SYSROOT}/usr/lib/firmware"
-EFI_DIR=/boot/efi/EFI/UpdateCapsule
+EFI_CAPSULE_DIR=/boot/EFI/UpdateCapsule
```

Comment:
does it need to use ESPMOUNT as part of the path?

Reply:
I'm not sure if we really need to pull that information from a different recipe (`[setup-nv-boot-control_1.0.bbappend](https://github.com/foundriesio/meta-lmp/pull/1533/files#diff-a9c1168f1d1cd22ecf857bc5af78c5cdc3c9887f2669aa99173c79d65cad3d84)`) so we can have a variable here instead of a magic string